test/motor_serial_test.cc

```diff
@@ -292,25 +292,25 @@ TEST_F(MotorSerialTests, writeMultipleOutputs) {
     std::vector<MotorMessage> commands;
 
     MotorMessage left_odom;
-    left_odom.setRegister(MotorMessage::REG_LEFT_ODOM);
+    left_odom.setRegister(MotorMessage::REG_BOTH_ODOM);
     left_odom.setType(MotorMessage::TYPE_READ);
     left_odom.setData(0);
     commands.push_back(left_odom);
 
     MotorMessage right_odom;
-    right_odom.setRegister(MotorMessage::REG_RIGHT_ODOM);
+    right_odom.setRegister(MotorMessage::REG_BOTH_ODOM);
     right_odom.setType(MotorMessage::TYPE_READ);
     right_odom.setData(0);
     commands.push_back(right_odom);
 
     MotorMessage left_vel;
-    left_vel.setRegister(MotorMessage::REG_LEFT_SPEED_MEASURED);
+    left_vel.setRegister(MotorMessage::REG_MAX_SPEED_FWD);
```

Comment:
Im not familier with the _SPEED_MEASURED values.   The change here does move to supported register names.